apps/challenges/utils.py

```diff
@@ -6,3 +8,9 @@
 get_challenge_model = get_model_object(Challenge)
 
 get_challenge_phase_model = get_model_object(ChallengePhase)
+
+
+def get_file_content(file_path, mode):
+    if os.path.isfile(file_path):
+        with open(file_path, mode) as file_content:
```

Comment:
I think this looks good for now and we can use `r` as default parameter later on.